doc/qbittorrent.1.md

```diff
@@ -1,36 +1,178 @@
+
+# ENVIRONMENT
+
+_OPTIONS_ may be supplied via environment variables with equivalent names. 
+
+For an option named **parameter-name**, the corresponding environment variable name is **QBT_PARAMETER_NAME**. In other words, the parameter name is converted to upper-case, any hyphens (**-**) are replaced by underscores (**\_**), and **QBT_** is prepended to it.
+
+To pass flag values, set the variable to **1** or **TRUE**. For example, to disable the splash screen: **QBT_NO_SPLASH=1**
 
-**`--no-splash`** Disables splash screen on startup.
+# FILES
 
-**`--webui-port=x`** Changes Web UI port to x (default: 8080).
+qBittorrent complies with the Freedesktop XDG Base Directory specification. TODO: not always, see "data" folder.
+Typically, **\$XDG_CONFIG_HOME** is _~/.config_ and **$XDG_DATA_HOME** is _~/.local/share_.
 
+TODO: did I miss any?
+
+_**$XDG_CONFIG_HOME**/qBittorrent/qBittorrent.conf_
+
+:   Main configuration file.
+
+_**$XDG_CONFIG_HOME**/qBittorrent/qBittorrent-data.conf_
+
+:   This is where all-time statistics are saved.
+
+_**$XDG_CONFIG_HOME**/qBittorrent/rss/feeds.json_
+
+:    RSS feeds configuration.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/BT_backup/_
+
+:    This directory contains the _.fastresume_ files, which is the mechanism qBittorrent uses to keep track of the state of each added torrent across restarts. In addition, a copy of each of the added torrents' meta-files is kept here, with the torrents' hashes as the file names.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/GeoIP/GeoLite2-Country.mmdb_
+
+:    Database for IP <-> country flag resolution in the "Peers" tab.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/logs_
+
+:    Execution logs.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/nova3/_
+
+:    This directory contains the Python files for the Search Engine functionality.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/nova3/engines/_
+
+:    Directory for custom search engine code and files.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/rss/articles_
+
+:    RSS feed storage.
+
+_**$XDG_CONFIG_HOME**/qBittorrent/rss/storage.lock_
+
+:    TODO: document lock files and their interaction with the configuration option.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/BT_backup/session.lock_
+
+:    TODO: document lock files and their interaction with the configuration option.
+
+_**$XDG_DATA_HOME**/data/qBittorrent/rss/articles/storage.lock_
+
+:    TODO: document lock files and their interaction with the configuration option.
+
+_/tmp/qtsingleapp-qBitto-1809-3e8-lockfile_
+
+:    TODO: document lock files and their interaction with the configuration option.
+
+# NOTES
+
+Refer to the wiki for addtional information, including guides on how to compile qBittorrent from source: https://github.com/qbittorrent/qBittorrent/wiki
 
 # BUGS
-If you find a bug, please report it at http://bugs.qbittorrent.org
+
+If you find a bug, please report it at https://bugs.qbittorrent.org
```

Comment:
bugs.qbittorrent.org doesn't serve HTTPS

Reply:
I can change that to https://github.com/qbittorrent/qBittorrent/issues if there are no objections or revert @Chocobo1, @glassez

Reply:
I believe @sledgehammer999 would like to keep the old one.

Reply:
bugs.qbittorrent.org is also referenced a lot across documentation so it wouldn't make sense to change it

Reply:
Reverted URL back to HTTP. It's better to fix the underlying issue anyway. When the URL is changed to HTTPS, we can change it here as well.